src/Infrastructure/Dashboard/TermMeta/InputTermMetaField.php

```diff
@@ -4,12 +4,16 @@
 
 namespace Fau\DegreeProgram\Infrastructure\Dashboard\TermMeta;
 
+use Fau\DegreeProgram\Application\FormFieldValidation\Rules;
+
 class InputTermMetaField implements TermMetaField
 {
+    use NoValidationTermMetaFieldTrait;
+
     public function __construct(
         private string $key,
-        private string $title,
-        private string $description = '',
+        protected string $title,
+        protected string $description = '',
```

Comment:
What's the background for these props upgrading to `protected` from `private`?

Reply:
This is because we are overriding these properties in the subclass.